docs/sandbox/Sandbox-FAQ.md

```diff
@@ -72,3 +72,12 @@ Currently not all of the securities we have set up in Production are mirrored in
 
 Similarly, we do not have up to date prices in Sandbox, and the ones that are generated each day stay the same for each ISIN.
 
+### Fees
+
+#### Sandbox
+
+Fee accrual, aggregation and charging is manual in sandbox, so if you want to test out fees, you'll need to contact your tenant manager. We will need the fee portfolio that you've set up and the accrual strategy details that you've agreed with your tenant manager. Typically we will simulate this for the previous month.
```

Comment:
```suggestion
Fee accrual, aggregation and charging is manual in sandbox. If you would like to test out fees in Sandbox please contact your tenant manager. 

To test in sandbox we will need the ID of the fee portfolio that you have set up and the accrual strategy details that have been agreed upon with your tenant manager. Typically we will simulate this for the previous month but let your tenant manager know if you want a different period.
```

Reply:
I wouldn't bother mentioning "aggregation" - the idea of fees being aggregated is foreign to our tenants, it's an implementation detail.
Just fee accrual and charging is fine.

Reply:
Is it worth mentioning that if they want the fees to be charged, they'll need to put enough cash in the portfolio themselves? (either via deposit or sell orders)